internal/activities/validate_structure.go

```diff
@@ -74,6 +74,22 @@ func (a *ValidateStructure) Execute(
 		failures = append(failures, extras...)
 	}
 
+	// Check that digitized SIPs only have one dossier in the content dir.
+	if params.SIP.Type == enums.SIPTypeDigitizedSIP {
+		entries, err := os.ReadDir(params.SIP.ContentPath)
+		if err != nil {
+			return nil, fmt.Errorf("ValidateStructure: check for unexpected dossiers: %v", err)
+		}
+
+		if len(entries) > 1 {
+			for _, entry := range entries[1:] {
+				failures = append(failures, fmt.Sprintf(
+					"Unexpected dossier: %s", entry.Name(),
+				))
+			}
+		}
```

Comment:
We don't know here which ones are the unexpected, if you want to show the directory names I'd show them all. Maybe "More than one dossier folder in the content directory: [list of directories]", or something like that. I also think not showing the list of directories is okay.